dist/bundle.js

```diff
@@ -1382,14 +1382,44 @@ Object.defineProperty(exports, 'doSetDraftTransactionAddress', {
     return _wallet.doSetDraftTransactionAddress;
   }
 });
-Object.defineProperty(exports, 'doSendSupport', {
+Object.defineProperty(exports, 'doSendTip', {
+  enumerable: true,
+  get: function get() {
+    return _wallet.doSendTip;
+  }
+});
+Object.defineProperty(exports, 'doWalletEncrypt', {
```

Comment:
@skhameneh 

I've noticed that after I use `yarn link`. Webpack starts to bring in functions that aren't used in this module. It was after I pointed `lbryinc` and `lbry-desktop` to my local instance of `lbry-redux`. Any idea how to prevent this?

Reply:
It'll be built against what's linked and it's subjective to your dev environment.
The proper fix is to remove artifacts from the repo.